website/docs/r/compute_instance_server.html.markdown

```diff
@@ -0,0 +1,134 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_compute_instance_server"
+sidebar_current: "docs-scaleway-resource-compute-instance-server"
+description: |-
+  Manages Scaleway Compute Instance servers.
+---
+
+# scaleway_compute_instance_server
+
+Creates and manages Scaleway Compute Instance servers. For more information, see [the documentation](https://developers.scaleway.com/en/products/instance/api/#servers-8bf7d7).
+
+## Examples
+
+### Minimal
+
+```hcl
+resource "scaleway_compute_instance_server" "web" {
+  type = "DEV1-S"
+  image_id = "f974feac-abae-4365-b988-8ec7d1cec10d"
+}
+```
+
+### With custom volumes and a public IP
+
+```hcl
+resource "scaleway_compute_instance_ip" "public_ip" {
+  server_id = "${scaleway_compute_instance_server.webserver.id}"
+}
+
+resource "scaleway_compute_instance_volume" "data" {
+  size_in_gb = 60
+}
+
+resource "scaleway_compute_instance_server" "web" {
+  type = "DEV1-L"
+  image_id = "f974feac-abae-4365-b988-8ec7d1cec10d"
+
+  tags = [ "hello", "public" ]
+
+  root_volume {
+    size_in_gb = 20
+  }
+
+  additional_volume_ids = [ "${scaleway_compute_instance_volume.data.id}" ]
+}
+```
+
+### With user data and could-init
+
+```hcl
+resource "scaleway_compute_instance_server" "web" {
+  type = "DEV1-L"
+  image_id = "f974feac-abae-4365-b988-8ec7d1cec10d"
+
+  tags = [ "web", "public" ]
+
+  user_data {
+    key = "plop"
+    value = "world"
+  }
+
+  user_data {
+    key = "xavier"
+    value = "niel"
+  }
+
+  cloud_init = file("${path.module}/cloud-init.yml")
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+- `type` - The server commercial type. You can use [this endpoint](https://api.scaleway.com/instance/v1/zones/fr-par-1/products/servers)
```

Comment:
add `(Required, force new resource)`

Reply:
at the end of the line: "Updates to this field will recreate a new resource." what do you think?